source/blog/_posts/2016-11-30-new-cart-bundle.md

```diff
@@ -0,0 +1,31 @@
+---
+title: New cart bundle concept
+tags:
+- cart
+- refactoring
+
+categories:
+- dev
+
+authors: [dr]
+
+github_link: blog/_posts/2016-11-30-new-cart-bundle.md
+---
+
+Since the last Community Day we get many questions about the new shopping cart:
```

Comment:
Since the last Community Day, we've received several questions about Shopware's new shopping cart: